plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -148,6 +149,25 @@ export function AboutCard(props: AboutCardProps) {
     }
   }
 
+  let techdocsHref: string | undefined;
+  if (viewTechdocLink) {
+    if (techdocsRef) {
+      techdocsHref = viewTechdocLink({
+        namespace: techdocsRef.namespace || DEFAULT_NAMESPACE,
+        kind: techdocsRef.kind,
+        name: techdocsRef.name,
+      });
+    } else {
+      techdocsHref = entity.metadata.annotations?.[TECHDOCS_ANNOTATION]
+        ? viewTechdocLink({
+            namespace: entity.metadata.namespace || DEFAULT_NAMESPACE,
+            kind: entity.kind,
+            name: entity.metadata.name,
+          })
+        : entity.metadata.annotations?.[TECHDOCS_REDIRECT_ANNOTATION];
```

Comment:
I think here is missing an update.
```suggestion
        : entity.metadata.annotations?.[TECHDOCS_LINKED_DOCS_ANNOTATION];
```